home-docs/code-style.md

```diff
@@ -1,9 +1,12 @@
 # Code style guidelines
 
 Import [project code style file](.github/checkstyle-configyration.xml) into IntelliJ to get correct formatting and ability to
-use "optimize imports" and "reformat code" actions. To do this you will need to
```

Comment:
Ref to [project code style file] returns responce with code 404 - page not found. Maybe should correct URI ?